packages/gatsby/src/schema/infer/example-value.js

```diff
@@ -0,0 +1,165 @@
+const _ = require(`lodash`)
+const { reportConflict } = require(`./type-conflict-reporter`)
```

Comment:
The current TypeConflictReporter exports a class, and the method is `addConflict` not `reportConflict`